### ISSUE TITLE
Base set of labels

### ISSUE THREAD

**Issue by SemenchenkoVitaliy** [metarhia/Metarhia#37]
When https://github.com/metarhia/tools/pull/5 lands we will need a base set of labels which will be used across all repositories of [`metarhia`](https://github.com/metarhia). I propose to discuss what labels we will need. These are some labels that we might use. Feel free to add, modify or remove them.

| name             | color  | description                                                                              |
|------------------|--------|------------------------------------------------------------------------------------------|
| ci               | ffff00 | PRs and issues related to the tests with unstable failures on the CI.                    |
| abandoned        | 000000 | when there is no eventual need                                                           |
| backport         | 4213c4 | Issues and PRs related to backporting features and bug fixes to previous versions.       |
| benchmark        | 009800 | PRs and issues related to the benchmark subsystem.                                       |
| blocked          | b60205 | PRs that are blocked by other issues/PRs.                                                |
| browser          | f9d0c4 | PRs and issues related to browser                                                        |
| bug              | fc2929 | PRs and issues related to                                                                |
| build            | eb6420 | PRs and issues related to build files or the CI.                                         |
| cli              | bfd4f2 | PRs and issues related to the Node.js command line interface.                            |
| coverage         | 76e85a | Issues and PRs related to native coverage support.                                       |
| deprecations     | ea8a8d | Issues and PRs related to deprecations.                                                  |
| deps             | 1da6db | Issues and PRs related to dependencies.                                                  |
| discussion       | 9fefc7 | Issues that are meant for discussion.                                                    |
| dist             | d4c5f9 | Issues and PRs related to dist folder.                                                   |
| doc              | 006b75 | Issues and PRs related to the documentation.                                            |
| duplicate        | ededed | Issues or PRs that are duplicates of other issues or PRs.                                |
| experimental     | c5def5 | Issues or PRs related to experimental features.                                          |
| feature-request  | 0d7bde | Issues that request new features to be added to the Node.js core.                        |
| good first issue | 7057ff | Issues that are suitable for first-time contributors.                                    |
| help wanted      | 159818 | Issues that need assistance from volunteers or PRs that need help to proceed.            |
| in progress      | d93f0b | Issues and PRs that are still in progress.                                               |
| install          | eb6420 | Issues and PRs related to the installers.                                                |
| invalid          | ededed | PRs and issues that are invalid.                                                         |
| investigating    | e407f4 | Issues and PRs that are not confirmed.                                                   |
| known limitation | ff8080 | Issues that are identified as known limitations.                                         |
| landed           | 6f42c1 | PRs that were landed.                                                                    |
| lib              | fad8c7 | Issues and PRs related to general changes in the lib directory.                          |
| lint             | 0e8a16 | Issues and PRs related to linter.                                                        |
| macos            | 9944dd | Issues and PRs related to the macOS platform.                                            |
| meta             | b60205 |                                                                                          |
| misspell         | fbca04 |                                                                                          |
| npm              | fad8c7 | Issues and PRs related to the npm client dependency or the npm registry.                 |
| optimization     | 008800 | Issues and PRs related to performance optimization.                                      |
| performance      | 5319e7 | Issues and PRs related to the performance of Node.js.                                    |
| question         | cc317c | Issues that look for answers.                                                            |
| regression       | b60205 | Issues related to regressions.                                                           |
| release          | 006b75 | Issues and PRs related to releases of new versions.                                      |
| research         | 0052cc | Issues and PRs related to research.                                                      |
| revisit later    | ff89ed | Inactionable items that could be revisited later                                         |
| security         | bb3344 | Issues and PRs related to security.                                                      |
| semver-major     | fbca04 | PRs that contain breaking changes and should be released in the next major version.      |
| semver-minor     | fef2c0 | PRs that contain new features and should be released in the next minor version.          |
| semver-patch     | f4f3c0 | PRs that contain new features and should be released in the next minor version.          |
| server           | e99695 | Issues and PRs related to server.                                                        |
| src              | 5319e7 | Issues and PRs related to general changes in the src directory.                          |
| stalled          | d4c5f9 | Issues and PRs that are stalled.                                                         |
| test             | 990099 | Issues and PRs related to the tests.                                                     |
| tools            | fe8a16 | Issues and PRs related to tools for development.                                        |
| triage-high      | 000000 |                                                                                          |
| triage-low       | 000000 |                                                                                          |
| triage-normal    | 000000 |                                                                                          |
| triage-paused    | 000000 |                                                                                          |
| windows          | 9944dd | Issues and PRs related to the Windows platform.                                          |
| wontfix          | ededed | Issues that will not be fixed, e.g. due to backwards compatibility concerns.             |
| work in progress | ffff00 | Issues and PRs that are still a work in progress.                                        |
| wrong repo       | 6a4636 | Issues that should be opened in another repository.                                      |
